src/Test.UnitTests.BinSkim.Driver/CompilerDataLoggerTests.cs

```diff
@@ -622,6 +631,139 @@ private void LogExceptionEvents(List<ExecutionException> exceptions, CompilerDat
             return sb;
         }
 
+        private StringBuilder ValidateChunkedEvents(string expectedEventName,
+                                                    List<ITelemetry> telemetryGeneratedEvents,
+                                                    string expectedChunkedContent,
+                                                    int chunkCount)
+        {
+            var sb = new StringBuilder();
+            string assemblyEventsId = string.Empty;
+            string commandLineEventsId = string.Empty;
+
+            var compilerEvents = new List<EventTelemetry>();
+            var assemblyReferencesEvents = new List<EventTelemetry>();
+            var commandLineEvents = new List<EventTelemetry>();
+            var summaryEvents = new List<EventTelemetry>();
+
+            foreach (EventTelemetry telemetryEvent in telemetryGeneratedEvents)
+            {
+                IDictionary<string, string> properties = telemetryEvent.Properties;
+
+
```

Comment:
there are 2 empty lines here, should be 1 only. I thought linter should mark it?

Reply:
Most other language, linters will flag. not in .NET.
:)